cookiecutter.json

```diff
@@ -6,6 +6,11 @@
     "monitoring": "y",
     "monitor_view_execution_time_in_djagno": "y",
     "monitor_tcpstat": "y",
+    "log_aggregating": "y",
+    "log_aggregator_url": "https://loki.reef.pl",
+    "log_aggregator_user": "acme",
+    "log_aggregator_client": "{{ cookiecutter.log_aggregator_user }}",
+    "log_aggregator_client_server_group": "{{ cookiecutter.log_aggregator_client }}_{{ cookiecutter.repostory_name }}",
```

Comment:
lets call these `monitoring_client*`
since these should be common for all monitoring tools

Reply:
But it is not for monitoring (Prometheus), it is only for log aggregation tool (Loki). If change the name, it would be misleading

Reply:
It kinda has to be same as in Prometheus tho or we will go insane, so IDK about it being misleading.

It would be also nice to use this value in sentry_sdk; all in effort to be able to somewhat correlate events across these different monitoring systems. But do as you will.

Reply:
Doesn't matter. It has to be removed anyway as we will not have cookiecutter.log_aggregator_user